### ISSUE TITLE
vim模式自动切换?

### ISSUE THREAD

**Issue by sunyueyin** [farazdagi/vim-go-ide#20]
当我输入个{},并把花括号另起一行时{
},按esc键,vim会自动从正常模式切换到插入模式,请问下怎么改成和正常的vim一样,找了好久没找到怎么改

**Comment by lylex** [comment]
How about using English to translate your question? So that you can get widely support.

**Comment by sunyueyin** [comment]
when i input a {} and then make the {} muti lines, my first esc cannot make my vim swith nomal status

**Comment by lylex** [comment]
Did you mean that, if you input { and then } in the next line in the insert mode, you have to try ESC more than once to turn back the the command mode? I tried with the latest vim-go-ide version, and seems good to me.

**Comment by sunyueyin** [comment]
yes,thanks,i will try it. now can you tell me what operate system when you do this,i do it at centos 6.5. and today i do it at macOs it not like that.

**Comment by lylex** [comment]
I am using ubuntu 16.04 and mac. Good to know that it works on one of your systems. Maybe, it is a bug on centos. @sunyueyin
